### ISSUE TITLE
More details about evaluation on traditional methods (e.g., SASRec)

### ISSUE THREAD

**Issue by mediumboat** [SAI990323/TALLRec#37]
Hi, this is really an interesting work. I have a question about the evaluation settings for traditional methods. 

When training and testing TALLRec, the historical interactions are constructed as the input prompt to LLM. I wonder in testing traditional methods, how these historical interactions are used. Do you further fine-tune the SASRec on the historical interactions in the test set or directly predict the test target item?

**Comment by jizhi-zhang** [comment]
Thank you very much for your attention to our work.

We predicted the target item directly and did not finetune the SASRec with the historical interactions of the test set.
The LLM was not trained on the input prompt of the test set, although the historical interaction was built into the input prompt when testing TALLRec. So we also didn't train with the historical interactions of the test set in traditional methods.

**Comment by mediumboat** [comment]
Thanks for your quick reply, it really helps
